XenAdmin/Wizards/NewNetworkWizard_Pages/NetWSriovDetails.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using XenAdmin.Controls;
+using XenAPI;
+using XenAdmin.Network;
+
+namespace XenAdmin.Wizards.NewNetworkWizard_Pages
+{
+    public partial class NetWSriovDetails : XenTabPage
+    {
+        internal Host Host;
+
+        public NetWSriovDetails()
+        {
+            InitializeComponent();
+        }
+        public override string Text { get { return Messages.NETW_DETAILS_TEXT; } }
+
+        public override string PageTitle
+        {
+            get
+            {
+                return Messages.NETW_INTERNAL_DETAILS_TITLE;
+            }
+        }
+        public override void PopulatePage()
+        {
+            PopulateHostNicList(Host, Connection);
+        }
+        private void PopulateHostNicList(Host host, IXenConnection conn)
+        {
+            comboBoxNicList.Items.Clear();
+            foreach (PIF ThePIF in conn.Cache.PIFs)

```

Comment:
Style recommendation: for local variables it's best to go with camelCase, not PascalCase, i.e. "thePif".

Reply:
Thanks, I have changed it.